pkgs/top-level/aliases.nix

```diff
@@ -402,6 +402,7 @@ mapAliases ({
   opencl-icd = ocl-icd; # added 2017-01-20
   openexr_ctl = ctl; # added 2018-04-25
   openjpeg_1 = throw "openjpeg_1 has been removed, use openjpeg_2 instead"; # added 2021-01-24
+  openjpeg_2 = openjpeg; # added 2021-01-25
   openjpeg_2_1 = openjpeg_2; # added 2018-10-25
```

Comment:
When can an alias be removed? This one should be since it actually points to version 2.4.

Reply:
Since it doesn't specify a minor revision, it could point to any of them. If it's not needed anymore, we could deprecate it anyway. The `openjpeg_2_1` alias, however, is clearly wrong and should be removed.

Reply:
I was referring to `openjpeg_2_1`.

Reply:
Oh wait, that was the one you pointed out. Nevermind :)